components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -1414,13 +1421,41 @@ - (void)setTrackingScrollView:(UIScrollView *)trackingScrollView {
   _shiftAccumulatorDeltaY = 0;
 
   _trackingInfo = [_trackedScrollViews objectForKey:_trackingScrollView];
+  _trackingInfo.stashedHeightIsValid = NO;
```

Comment:
When setting this to `NO` should we reset it in some way?
`_trackingInfo.stashHeight = -1;`